.github/workflows/release.yml

```
@@ -9,7 +9,7 @@ on:
 jobs:
   release:
     runs-on: ubuntu-latest
-    if: github.repository == 'hashintel/hash'
+    if: github.repository == github.repository
```

Why this change? It will always be true, releasing to npm isn't going to work on forks, and it'll only run on merge to `main` in any case.

---

Whops, that was a search/replace error (I used `'dashintel/hash/'` previously but replaced it with `GitHub.repository`.
```suggestion
    if: github.repository == 'hashintel/hash'
```